### DISCUSSION TITLE
Support for anonymous sign in?

### DISCUSSION THREAD

**Discussion by happy-san** [supabase/supabase#683] [Feature Requests] [answered]
I understand that Supabase is not meant to be 1:1 replica of firebase, but I quite like the feature of letting users anonymously enter into the application, and then if they want to continue, they can upgrade their account.

**Comment by kiwicopple** [comment, marked as answer]
It's a good suggestion @happy-san . Do you want to add it to the GoTrue repo? 

https://github.com/supabase/gotrue

**Reply by happy-san** [reply]
As an issue?

**Reply by kiwicopple** [reply]
Yup! Thanks

**Reply by jojomatik** [reply]
Link to the issue https://github.com/supabase/gotrue/issues/68
